.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 defaults: &defaults
   working_directory: ~/go/src/github.com/docker/libnetwork
   docker:
-    - image: 'circleci/golang:1.10'
+    - image: 'circleci/golang:1.10.2'
```

Comment:
this image is irrelevant, everything is built within a container, we can probably take something smaller also

Reply:
The image we use comes from here: https://github.com/circleci-public/circleci-dockerfiles.   There isn't a minimal image in that repo.    We could try https://hub.docker.com/r/circleci/slim-base/ but I don't know if it's actively maintained.    CircleCI requires a few utilities in the image https://circleci.com/docs/2.0/custom-images so we could make an Alpine-based image, but then we would have to maintain it.